client/test/router_store.spec.js

```diff
@@ -1460,29 +1367,44 @@ describe('RouterStore', () => {
     // });
 
     it('REMOVE_ACCOUNT_SUCCESS', () => {
-      const accounts = AccountStore.getAll();
+      accounts = AccountStore.getAll();
       const accountIDs = _.keys(accounts.toJS());
+      const firstAccount = accountIDs.shift();
+      const secondAccount = accountIDs.shift();
+      const thirdAccount = accountIDs.shift();
+
+      // while we are editing the second account
+      Dispatcher.dispatch({
+        type: ActionTypes.ROUTE_CHANGE,
+        value: {
+          action: AccountActions.EDIT,
+          accountID: secondAccount },
+      });
 
       // No changes
       Dispatcher.dispatch({
         type: ActionTypes.REMOVE_ACCOUNT_SUCCESS,
-        value: { accountID: accountIDs.shift(), silent: true },
+        value: { accountID: firstAccount, silent: true },
       });
-      assert.equal(RouterStore.getAction(), null);
+
+      // ROMAINEDIT : action is never null
```

Comment:
Don't understand this comment :(
Any way the success/error redirection should be tested; it is failing (and wasn't weeks ago before Redux migration)